docs/DeveloperGuide.adoc

```diff
@@ -236,11 +236,12 @@ This section describes some noteworthy details on how certain features are imple
 
 The undo/redo mechanism is facilitated by an `UndoRedoStack`, which resides inside `LogicManager`. It supports undoing and redoing of commands that modifies the state of the address book (e.g. `add`, `edit`). Such commands will inherit from `UndoableCommand`.
 
-`UndoRedoStack` only deals with `UndoableCommands`. Commands that cannot be undone will inherit from `Command` instead. The following diagram shows the inheritance diagram for commands:
+`UndoRedoStack` only deals with `UndoableCommands`. Commands that cannot be undone will inherit from `Command` instead. The diagram below (Figure 11) shows the commands inheritance:
```

Comment:
`The following diagram (Figure 11) shows the inheritance diagram for commands` is better

Reply:
Noted with thanks 👍